src/NetAnalyzers/UnitTests/Microsoft.CodeQuality.Analyzers/ApiDesignGuidelines/AvoidOutParametersTests.cs

```diff
@@ -476,12 +476,25 @@ public class InterfaceBothImplicitAndExplicitImpl : IInterface
         throw new System.NotImplementedException();
     }
 }
-");
-            await VerifyVB.VerifyAnalyzerAsync(@"
+";
+            await new VerifyCS.Test
+            {
+                TestState =
+                {
+                    Sources = { csSource },
+                    AnalyzerConfigFiles = { ("/.editorconfig", @"root = true
+
+[*]
+dotnet_code_quality.CA1021.api_surface = all
+") },
+                }
+            }.RunAsync();
+
+            var vbSource = @"
 Imports System.Runtime.InteropServices
 
 Interface IInterface
-    Sub InterfaceMethod(<Out> ByRef s As String)
+    Sub [|InterfaceMethod|](<Out> ByRef s As String)
```

Comment:
So, the difference between this test and the C# test prior to this PR was that the interface was marked public for the C# test and is internal for VB test. I think we should keep both C# and VB tests in this test consistent - either both should declare the interface as internal and have the editorconfig setting `all` for `api_surface` OR both should declare the interface as public and not require any editorconfig setting.

Reply:
@mavasani editorconfig setting is needed either way to make sure we're testing the explicit interface implementation which is private. I'll update the VB test to match C# for consistency.